### DISCUSSION TITLE
BIRT Line chart X axis issue when executed in different Language Windows operating system

### DISCUSSION THREAD

**Discussion by raviprsd18** [eclipse-birt/birt#1600] [Q&A] [answered]
Dear BIRT Team,

I am facing the following issue, any help is highly appreciated.

**Problem Statement**: I have a Line chart grouped by date series in Y optional axis, X axis should always show 24 hours separated by 15 minutes as interval ( Attached the image )
                   The graph should be plotted against X axis 15 minutes interval along with Y axis value

**Issue**: The requirement works perfectly in English language Windows Operating system, Indian time zone (IST).
       But have different output in German language Windows Operating system, CET time zone. The X axis is displayed till **17.30** only because there is a time difference of **4 hours and 30 minutes** between IST and CET time zone.

**English language Windows Operating system, Indian time zone (IST)**
![lineChart_English_OS_IST_TimeZone](https://github.com/eclipse-birt/birt/assets/46317080/970ebad9-c1a2-4492-85e5-59ec447dea4f)

**German language Windows Operating system, CET time zone**
![lineChart_German_OS_CET_TimeZone](https://github.com/eclipse-birt/birt/assets/46317080/362bb58e-63e4-4a23-aa23-40abc592f816)

	   
Is there any way to fix the chart X axis 15 minutes interval to 24 hours irrespective of time zone and operating system.

**Note**: Refer the image on setting the 15 minutes interval in the line chart
![Steps_ToSet_15MinInterval_In_Xaxis](https://github.com/eclipse-birt/birt/assets/46317080/eaee3e53-7813-4ff2-87c3-7587ba1ca446)

Attached the rptDesign file for further troubleshooting.
[15MinInterval.zip](https://github.com/eclipse-birt/birt/files/14645873/15MinInterval.zip)

**Comment by hvbtup** [comment]
Without actually investigating this, to me this looks like a bug, either in BIRT or in your report.
Probably there's kind of a loop for the dates, and a comparison to an end date, and I guess that the end date is calculated wrong, with a different time zone or something similar.
Can you provide a simple test report which others can test?

**Reply by raviprsd18** [reply]
Hi @hvbtup, 
I have already attached the rptDesign in my thread. It is present inside the zip folder.

**Reply by raviprsd18** [reply]
@hvbtup, by any chance you got time to look into the issue ?

**Reply by hvbtup** [reply]
No, unfortunately not.

**Comment by raviprsd18** [comment]
@speckyspooky, any help here is highly apricated.

**Comment by raviprsd18** [comment, marked as answer]
The issue got resolved, by checking the check box _**Is Category Axis**_ for X axis, and we need to simulate the dataset in such a way the X axis should contain all the 15 intervals for 24 hours in the dateTime column.
